processor/metricstransformprocessor/config.go

```diff
@@ -112,6 +112,9 @@ const (
 	// Update updates an existing metric.
 	Update ConfigAction = "update"
 
+	// ChangeDataType changes the data type from int64 to double, or vice-versa
+	ChangeDataType OperationAction = "change_data_type"
```

Comment:
Basing off of our discussion, I would also suggest changing this operation action name to `toggle_data_type`, since at this point, there are only 2 possible values.